src/RelayPublic.js

```diff
@@ -47,7 +46,8 @@ var RelayPublic = {
   createContainer: RelayContainer.create,
   createQuery: createRelayQuery,
   getQueries: getRelayQueries,
-  injectNetworkLayer: RelayNetworkLayer.injectNetworkLayer,
+  injectNetworkLayer: RelayInternals.NetworkLayer.injectNetworkLayer
```

Comment:
`RelayInternals` is meant for exporting some functions for dev tools - to reference a module, import it normally. In this case something alone the lines of `RelayStoreData.getDefaultInstance().getNetworkLayer().bind(...)`